events/2022-january-15/group_agenda.md

```diff
@@ -0,0 +1,95 @@
+#  Jupyter Accessibility Workshops with Frank Elvasky: How to build inclusive data representations
+
+## Event details
+
+Date: January 15, 2022
+
+Overview: A friendly virtual event that brings Jupyter and PyData community members together to learn from an accessibility expert. 
+
+## Takeaways
+
+Jupyter and PyData members will leave better informed and more empathetic to accessibility considerations in their work while the accessibility community will become familiar with Jupyter and its open-source challenges.
+
+## What you need to participate
+
+* Some familiarity with data/visualization skills
+
+## Schedule
+
+```mermaid
+gantt
+
+    dateFormat YYYY-MM-DD hh:mm
+    axisFormat  %H:%M
+
+    section icebreaker
+    meet and greet :meet, 2021-10-02 08:45, 15min
+    welcome, welcome, welcome :welcome, after meet, 15min
+        
+    section introductions
+    intro to jupyter :jupyter, after welcome, 10min 
+    jupyterlab accessibility :ja11y, after jupyter, 10min
+    ally 101 :a101y, after ja11y, 15min
+        
+    section main 
+    main speaker :main, after a101y, 35min
+    questions :qa, after main, 25min
+    
+    
+    section breakouts
+    breakout topics :topics, after qa, 5min
+    break :break, after topics, 5min
+    breakout 1 :b1, after topics, 30min
+    breakout 2 :b2, after topics, 30min
+    summarize :regroup, after b1, 10min
+    breakout 1 :b21, after regroup, 25min
+    breakout 2 :b22, after regroup, 25min
+        
+    section conclusion
+    final comments :end, after b21, 15min
+    
+    section hangout (optional)
+    hangout :hang, after end, 45min
+    
+    
+```    
+
+* Enter in breakout rooms (like you're about to go on an amusement park ride)
+* Welcome and set goals
+    * To educate and introduce the Jupyter community to accessibility concerns 
+    in software.
+    * Connect the Jupyter community to accessibility experts and disabled 
+    people.
+    * Learn about web content accessibility through expert content and group 
+    discussions.
+* Intro to Jupyter and Jupyterlab accessibility
```

Comment:
I think this section might break the flow because summarizing current Jupyter accessibility efforts is a larger scope than the speaker's topic.

I also think we run the risk of repeating things the speaker might have to say since we don't know exactly everything that's in that talk yet.

```suggestion
```

Reply:
Added for now. It's always easier to remove if we decide to do so.

The only thing is I don't think it's wise to start a collaborative discussion right before a speaker in order to make sure we stay on time. But I do think this is a good idea to prime people for taking in this wider knowledge.